themes/default/content/learn/_index.md

```diff
@@ -22,17 +22,19 @@ tutorials:
         Tutorials are series of hands-on activities that help you along your
         learning journey with Pulumi.
 
+# There are 2 trailing spaces appended to each line in the paragraphs below
+# in order to force a new line to limit the line length to 75 characters.
```

Comment:
I modified this to be 75 characters/line. I would have liked to do this with CSS, but the content for those paragraphs is being piped into markdownify when we render and  whatever "magic" it is doing seems to have styling that is overriding what I was trying to do and kept messing with the text alignment. So I ended up just adding new lines in the actual markdown.